packages/keyring-controller/src/KeyringController.ts

```diff
@@ -677,7 +680,8 @@ export class KeyringController extends BaseController<
   }
 
   /**
-   * Update keyrings in state and calls KeyringController fullUpdate method returning current state.
+   * Update keyrings and vault in state and calls KeyringController
+   * fullUpdate method returning current state.
```

Comment:
Nit: Revert this description

Reply:
Oh right! I just updated it along with the PR description